java-persistence/src/main/java/com/whitecloak/training/inventory/controller/response/ItemResource.java

```diff
@@ -1,10 +1,16 @@
 package com.whitecloak.training.inventory.controller.response;
 
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.whitecloak.training.inventory.model.Category;
+import com.whitecloak.training.inventory.model.User;
+
+@JsonSerialize
 public class ItemResource {
 
     private long id;
     private String name;
-    private long ownerId;
+    private Category category;
+    private User owner;
```

Comment:
Resources are supposed to be composed with resources.

Instead of composing `ItemResource` with `Category` and `User`, you should have used `CategoryResource` and `UserResource`.